### PR TITLE
update return error for buy_fee_asset

### PR DESCRIPTION
**Pull request by mzxyz**
* update return error for buy_fee_asset, will use error info directly from `cennzx_spot::buy_fee_asset`

After export the specific error message, I could continue working on reasonable error msg in Transaction Payment

### CODE REVIEW

File: runtime/tests/tests.rs [hunk 1]
```diff
@@ -25,7 +25,6 @@ use frame_support::{additional_traits::MultiCurrencyAccounting, traits::Imbalanc
 use sp_runtime::{
 	testing::Digest,
 	traits::{Convert, Header},
-	transaction_validity::InvalidTransaction,
```
Comment (cowboy-bebug) [inline runtime/tests/tests.rs]: Could add this line back so it doesn't create conflicts with our PR lol

**Review by hoani — APPROVED** [review]
LGTM

**Review by cowboy-bebug — APPROVED** [review]
LGTM !

**Review by jordy25519 — APPROVED** [review]
👍

**Pull request merged**